packages/chat-stateful-client/src/ChatClientState.ts

```diff
@@ -88,6 +88,20 @@ export type ChatThreadProperties = {
   topic?: string;
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   createdBy?: CommunicationIdentifierKind;
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  messagingPolicy?: MessagingPolicy;
+};
+
+/* @conditional-compile-remove(rich-text-editor-image-upload) */
+/**
+ *
+ * Messaging policy of a chat thread.
+ *
+ * @beta
+ */
+export type MessagingPolicy = {
+  /** Boolean to track whether or not messages are restricted to only text. */
+  textOnlyChat?: boolean;
```

Comment:
This is a mirror of the underlying ACS Chat SDK value, exported explicitly to avoid linkage, correct?

Reply:
Yes, this is a mirrow of [this one](https://learn.microsoft.com/en-us/javascript/api/%40azure/communication-chat/chatthreadproperties?view=azure-node-preview#@azure-communication-chat-chatthreadproperties-messagingpolicy)